learning_flexbox/coll.scss

```diff
@@ -0,0 +1,117 @@
+* {
+    box-sizing: border-box;
+    padding: 0rem;
+    margin: 0rem;
+}
+
+body {
+    background-color: $base-blue;
+}
+
+.overlay {
+    position: fixed;
+    width: 100%;
+    height: 100%;
+    background-color: $base-blue;
+    opacity: 0.2;
+}
+
+.container>div {
+    min-width: 20rem;
+    min-height: 30rem;
+}
+
+.container {
+    display: grid;
+    grid-template-columns: repeat(auto-fit, minmax(20rem, 1fr));
+    grid-auto-rows: 30rem;
+    grid-auto-flow: dense;
+}
+
+.big {
+    grid-column: span 2;
+    grid-row: span 2;
+}
+
+
+.img1 {
+    background-image: url(images/1image.jpg);
```

Comment:
Remember what I said a few weeks ago? `background-image` should not be used in a CSS file for performance reasons. The page won't load unless the CSS AND the image are fully loaded. That will cause a webpage to take too long to be loaded. Prefer to use `<img>`, or adding the CSS inline: `<div style="background-image: url(images/1image.jpg)">`